modulefiles/hera.intel/coupled

```diff
@@ -15,7 +15,8 @@ setenv NCEPLIBS /scratch2/NCEPDEV/nwprod/NCEPLIBS
 ## load contrib environment
 ## load slurm utils (arbitrary.pl  layout.pl)
 ##
-module load contrib sutils
+module use -a /contrib/sutils/modulefiles
+module load sutils
```

Comment:
I thought this change were already committed by Bin, I am curious why this is shown again?

Reply:
I don't know. I've merged both my develop and this branch w/ that change but for some reason it keeps showing me being one commit behind the develop branch.